docs/mobile/events.md

```diff
@@ -21,46 +20,57 @@ application lifecycle. This event is meant to be used in conjunction with
 `os.name` [resource semantic convention](/docs/resource/os.md) to identify the
 mobile operating system (e.g. Android, iOS).
 
-### iOS
+Payload attributes MUST be used to describe the state of the application at the
+time of the event. The following table describes the payload attributes that MUST
+be used to describe the state of the application at the time of the event.
 
-<!-- semconv ios.lifecycle.events -->
-The event name MUST be `device.app.lifecycle`.
-
-| Attribute  | Type | Description  | Examples  | Requirement Level |
-|---|---|---|---|---|
-| `ios.state` | string | This attribute represents the state the application has transitioned into at the occurrence of the event. [1] | `active` | Required |
+The `android.state` and `ios.state` attributes are mutually exclusive and MUST
+NOT be used together, each attribute MUST be used with it's corresponding
+`os.name` [resource semantic convention](/docs/resource/os.md) value. For
+example, if the `os.name` attribute is set to `android` then the
+`android.state` attribute MUST be used and the `ios.state` attribute MUST NOT
+be used. If the `os.name` attribute is set to `ios` then the `ios.state`
+attribute MUST be used and the `android.state` attribute MUST NOT be used.
 
-**[1]:** The iOS lifecycle states are defined in the [UIApplicationDelegate documentation](https://developer.apple.com/documentation/uikit/uiapplicationdelegate#1656902), and from which the `OS terminology` column values are derived.
+### Event details
 
-`ios.state` MUST be one of the following:
+<!-- semconv device.app.lifecycle(full) -->
+The event name MUST be `device.app.lifecycle`.
```

Comment:
not related to this PR, but it'd be nice to render a table here (see https://github.com/open-telemetry/build-tools/issues/292)